backend/user/views.py

```diff
@@ -109,14 +119,32 @@ def post(self, request):
                 notificationsetting__enabled=True,
             )
             .exclude(token="")
+            .values_list("user__username", "token", "dev")
         )
 
+        # unpack list of tuples
+        if tokens:
+            success_users, tokens, devs = zip(*tokens)
+        else:
+            success_users, devs = list(), list()
+        failed_users = list(set(usernames) - set(success_users))
+
         if len(tokens) == 1:
```

Comment:
hmm, I know we used this everywhere, but it is getting a little confusing with all of the `tokens` variables around. I think this one refers to the `tokens` variable after the zip, and the one on L126 refers to the QuerySet. Do you think we should change or nah?